packages/db/src/translate/async_operations.ts

```diff
@@ -10,7 +10,7 @@ export const translateApiAsyncOperationToPostgresAsyncOperation = (
   if (record.output === 'none') {
     delete translatedRecord.output;
   } else if (record.output !== undefined) {
-    translatedRecord.output = JSON.parse(record.output);
+    translatedRecord.output = JSON.parse(JSON.stringify(record.output));
```

Comment:
Interesting that knex isn't casting objects properly in this instance.   

I spent a bit of time trying to decide if the possibility of some of our un-typescripted code/user passing in an object (in violation of type) that results in something other than a write failure and/or a reasonable edge case and could not.    Commenting in case that shakes anything loose, but going to approve.